apps/docs/src/.vitepress/components/ComponentGrid.vue

```diff
@@ -0,0 +1,78 @@
+<script lang="ts" setup>
+import { computed } from "vue";
+import ComponentCard, { type ComponentCardProps } from "./ComponentCard.vue";
+
+export type ComponentGridProps = {
+  /** Description to show above the grid. */
+  description: string;
+  /** List of components to show. */
+  components: ComponentCardProps[];
+  /** Timestamp of estimated due date (will be formatted as month + year). */
+  dueDate?: string;
+};
+
+const props = defineProps<ComponentGridProps>();
+
+const formattedDueDate = computed(() => {
+  if (!props.dueDate) return;
+  const date = new Date(props.dueDate);
+  return Intl.DateTimeFormat("en-US", {
+    month: "long",
+    year: "numeric",
+  }).format(date);
+});
+
+/**
+ * Sorts components by: (1.) implemented first and (2.) alphabetically.
+ */
+const sortedComponents = computed(() => {
+  return props.components
+    .slice()
+    .sort((a, b) => a.name.localeCompare(b.name))
+    .sort((a, b) => {
+      if (a.implemented && !b.implemented) return -1;
+      if (!a.implemented && b.implemented) return 1;
+      return 0;
+    });
+});
+</script>
+
+<template>
+  <div class="content">
+    <div class="description">
+      <p>{{ props.description }}</p>
+      <p class="description__date">
+        Estimated due date: {{ formattedDueDate ?? "Not planned yet" }}
+      </p>
+    </div>
+
+    <div class="grid">
+      <ComponentCard
+        v-for="component in sortedComponents"
+        :key="component.name"
+        v-bind="component"
+      />
+    </div>
+  </div>
+</template>
+
+<style lang="scss" scoped>
+.content {
+  padding: 1rem 2rem 2rem;
+}
+
+.description {
+  color: var(--vp-c-text-2);
+  margin-bottom: 2rem;
```

Comment:
same argument as above
```suggestion
  margin-bottom: 1.5rem;
```